src/field_theory/adjoin.lean

```diff
@@ -644,6 +644,20 @@ begin
   exact induction_on_adjoin_fg P base ih K K.fg_of_noetherian
 end
 
+instance alg_hom.finite_of_finite_dimensional [finite_dimensional F E] : finite (E →ₐ[F] K) :=
```

Comment:
Isn't this just a weaker version of [field.alg_hom.fintype](https://leanprover-community.github.io/mathlib_docs/field_theory/primitive_element.html#field.alg_hom.fintype)? Can't `apply_instance` find that?

Reply:
Whcih itself looks like it might be a duplicate of [`alg_hom.fintype`](https://leanprover-community.github.io/mathlib_docs/field_theory/fixed.html#alg_hom.fintype)?

Reply:
Ah, yes, it is a duplicate of `field.alg_hom.fintype`. I just need to import `primitive_element.lean`

`alg_hom.fintype` assumes finite-dimensionality of the target.

Reply:
Thanks!

Reply:
So is `alg_hom.fintype` strictly weaker than `field.alg_hom.fintype`?

Reply:
Sure. Why not just replace the weaker version by the stronger version? #15345